src/constants/env.ts

```diff
@@ -423,7 +429,7 @@ function parseEnv(env: Env): Config {
           chainId: parseInt(chainId),
           address: contractConfig.address,
           deployBlock: contractConfig.deployBlock,
-          blockHistoryLimit: 100000,
+          blockHistoryLimit: 1000000,
```

Comment:
default block limit is higher now. this may affect other things so keep an eye out if something breaks